cnd/src/network/mod.rs

```diff
@@ -1,10 +1,14 @@
 pub mod transport;
 
+use crate::swap_protocols::rfc003::create_swap::SwapEvent;
```

Comment:
:mega: Put these in the same block as the others please so the imports get merged.